src/main/java/frc/robot/subsystems/drive/Drive.java

```diff
@@ -61,6 +62,9 @@ public Drive(
             RobotConstants.get().maxLinearSpeed(),
             RobotConstants.get().moduleTranslations()[0].getNorm(),
             new ReplanningConfig()),
+        () -> {
+          return DriverStation.getAlliance().get() == DriverStation.Alliance.Red;
```

Comment:
this could cause a NullPointerException, please check if it is present before calling it. maybe default to false.